cfg/gke-1.4.0/managedservices.yaml

```
@@ -0,0 +1,706 @@
+---
+controls:
+version: "gke-1.4.0"
+id: 5
+text: "Managed Services"
+type: "managedservices"
+groups:
+  - id: 5.1
+    text: "Image Registry and Image Scanning"
+    checks:
+      - id: 5.1.1
+        text: "Ensure Image Vulnerability Scanning using GCR Container Analysis
+        or a third-party provider (Manual)"
```

```suggestion
        or a third-party provider (Automated)"
```
https://workbench.cisecurity.org/sections/1652562/recommendations/2654516